import-export-cli/cmd/mg/mg.go

```diff
@@ -42,5 +42,5 @@ var MgCmd = &cobra.Command{
 }
 
 func init() {
-	RootCmd.AddCommand(MgCmd)
+	// cmd.RootCmd.AddCommand(MgCmd)
```

Comment:
Shall we remove this whole **init()** function or is there a reason for keeping it (a future purpose perhaps)?